src/transformStateWithClones.js

```diff
@@ -7,7 +7,43 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const reducer = (reducerState, action) => {
```

Comment:
Would be nice to move this helper function outside of `transformStateWithClones` function to make it reusable